src/runtime/test/api-channel-test.ts

```diff
@@ -7,68 +7,119 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-'use strict';
 
 import {assert} from '../../platform/chai-web.js';
+import {Arc} from '../arc.js';
 import {APIPort, PECOuterPort, PECInnerPort} from '../api-channel.js';
 
 class Registrar {
+  // tslint:disable-next-line: no-any
+  calls: Map<string, any> = new Map();
+  // tslint:disable-next-line: no-any
+  handlers: Map<string, any> = new Map();
+  // tslint:disable-next-line: no-any
+  initializers: Map<string, any> = new Map();
+  // tslint:disable-next-line: no-any
+  initializerHandlers: Map<string, any> = new Map();
+
   constructor() {
-    this.calls = new Map();
-    this.handlers = new Map();
-    this.initializers = new Map();
-    this.initializerHandlers = new Map();
   }
 }
 
 // TODO(shans): Make this test work with the new API channel
+// tslint:disable-next-line: only-arrow-functions
 describe('API channel', function() {
   let outer;
   let inner;
 
   before(() => {
     APIPort.prototype._testingHook = function() {
       // Change the argumentType mapping objects/functions to generate string identifiers.
-      this.Direct = 'Direct';
```

Comment:
This seems brittle, can a TODO be left to change this to use a dependency injection, rather than overriding parameters of this (which I think is the APIPort in this case)?

Reply:
Added a TODO.  I think there might be a more idiomatic way of implementing testingHook here, but wanted to just focus on getting this into typescript for now.